Source/core/WarningReportingControl.h

```diff
@@ -146,6 +147,30 @@ namespace Core {
 
 namespace WarningReporting {
 
+    class EXTERNAL ExcludedWarnings {
```

Comment:
Better make it consistent with our other code (we alwyas add constructor (not defaulkt), destructor, (deleted) copy and assignment operators)